pdudaemon/drivers/ykush.py

```diff
@@ -24,26 +25,27 @@
 import logging
 from pdudaemon.drivers.driver import PDUDriver
 import hid
-
 import os
+
 log = logging.getLogger("pdud.drivers." + os.path.basename(__file__))
 
 YKUSH_VID = 0x04d8
+YKUSH_PID = 0xf2f7
 YKUSH_XS_PID = 0xf0cd
 
 
-class YkushXS(PDUDriver):
+class YkushBase(PDUDriver):
     connection = None
-    port_count = 2
-    supported = ["YKUSHXS"]
+    ykush_pid = None
+    port_count = 0
 
     def __init__(self, hostname, settings):
         self.hostname = hostname
         self.settings = settings
         self.serial = settings.get("serial", u"")
         log.debug("serial: %s" % self.serial)
 
-        super(YkushXS, self).__init__()
+        super(YkushBase, self).__init__()
```

Comment:
Since its Python3 code, you can just do super().__init__() and copy/paste gets more fool-proof

Reply:
good hint, thanks! I'll wait for the review from @sjoerd-ccu and update